src/measure_theory/interval_integral.lean

```diff
@@ -1005,6 +1009,14 @@ integral_mono_ae hab hf hg $ ae_of_all _ h
 
 end mono
 
+lemma abs_integral_le_integral_abs (h : interval_integrable f μ a b) :
```

Comment:
The integrability assumption is probably not necessary here (if the function is not integrable, the left hand side is zero and the result is true anyway). Also, you could maybe state and prove a version with norms, and then deduce the version with abs?

Reply:
I had originally tried to prove this lemma for norms but didn't succeed. The fact that we have [`interval_integral.norm_integral_le_abs_integral_norm`](https://leanprover-community.github.io/mathlib_docs/measure_theory/interval_integral.html#interval_integral.norm_integral_le_abs_integral_norm) made me think that it perhaps is not actually true. Is that not so?

Reply:
Indeed, this lemma contains the fact you need, so you should prove your lemma using this one. The difference is that the other lemma does not assume `a ≤ b` while yours does. If you assume it, the integral of a nonnegative function is nonnegative, and you can get rid of the additional `abs`.